hack/build/push_image.sh

```diff
@@ -7,24 +7,16 @@ if [[ ! "${TAG}" ]]; then
 fi
 
 commit=$(git rev-parse HEAD)
-build_date="$(date -u +"%Y-%m-%dT%H:%M:%S+00:00")"
-go_build_args=(
-  "-ldflags=-X 'github.com/Dynatrace/dynatrace-operator/src/version.Version=${TAG}'"
-  "-X 'github.com/Dynatrace/dynatrace-operator/src/version.Commit=${commit}'"
-  "-X 'github.com/Dynatrace/dynatrace-operator/src/version.BuildDate=${build_date}'"
-  "-s -w"
-)
+go_build_args=$(hack/build/ci/hack/build/ci/create_go_build_args.sh "${TAG}" "${commit}")
 base_image="dynatrace-operator"
 out_image="quay.io/dynatrace/dynatrace-operator:${TAG}"
 
-
-args="${go_build_args[*]}"
 if [[ "${LOCALBUILD}" ]]; then
   export CGO_ENABLED=1
   export GOOS=linux
   export GOARCH=amd64
 
-  go build "$args" -o ./build/_output/bin/dynatrace-operator ./src/cmd/operator/
+  go build "$go_build_args" -o ./build/_output/bin/dynatrace-operator ./src/cmd/operator/
```

Comment:
`push_image` shouldn't build the image, but that's a different problem.
since `ci/create_go_guild_args` is not only used for ci, it should be moved one folder up.

Reply:
agree

moved